### PR TITLE
[Bluesky] Add followers and posts badges

### LINKED ISSUE #10788: Badge Request - Dynamic Bluesky Badges for Users
### 📋 Description

Mostly as another variation on a social follower/post count badges similar to other services.

Utilizing the static generator for a personal examples:

Possible follower example: Static Badge
Possible post count example: Static Badge

I can see some use with a badge for a specific feed as well, but I have not looked into this matter deeply enough to give a confident use case.

### 🔗 Data

The Bluesky API for this data is publicly available and accessible with no API keys.

The documentation is available on docs.bsky.app as [app-bsky-actor-get-profile](https://docs.bsky.app/docs/api/app-bsky-actor-get-profile).

It seems that just using `https://public.api.bsky.app/xrpc/app.bsky.actor.getProfile/?actor=[HANDLE / DID]` is all that's needed for the required data as JSON, accessible with the keys `followersCount` and `postsCount` for the examples I gave above. The usage of the `handle` key can be used as the username of the account in the badge.

Only main issue I can foresee is handles changing, but that can be resolved by using the DID of the user, which for convenience can be captured at the time with the `did` key.

### 🎤 Motivation

More social badges would be helpful, especially for people who operate either solely or majorly on Bluesky over other services. I can see this being used on GitHub accounts' README.md as a social media tie, and generally used as a way to encourage support of developers and projects outside of GitHub, Bitbucket, any other Git repository service, or webpage in general.

As a personal motivation, Bluesky is my only main social media platform I actively use, but I know that the developer presence on Bluesky is generally substantial.

### PR DESCRIPTION
**Pull request by chitvs**
This PR adds two new badges for the Bluesky social network:

- Followers badge: displays the number of followers a user has.
- Posts badge: displays the number of posts a user has made.

Both badges are implemented with proper schema validation and corresponding tests. This contributes to the social category of Shields.io badges.

Fixes #10788

### CODE REVIEW

**Comment by PyvesB** [comment]
What is the rate limit for the https://public.api.bsky.app/xrpc/app.bsky.actor.getProfile endpoint?

**Comment by chitvs** [comment]
> What is the rate limit for the https://public.api.bsky.app/xrpc/app.bsky.actor.getProfile endpoint?

The https://public.api.bsky.app endpoint has generous rate limits according to [this](https://docs.bsky.app/docs/advanced-guides/rate-limits).

**Comment by PyvesB** [comment]
Sounds like we're good on the rate limiting side!

I'm guessing you probably used an old PR as an example to put these new badges together. Unfortunately, a few things have changed since then, in particular how we build examples (see #9285). You'll need to update those bits. I'd suggest either referring to [our tutorial](https://github.com/badges/shields/blob/master/doc/TUTORIAL.md) for up-to-date documentation, or using one of our existing social services as inspiration. 😉

**Comment by chitvs** [comment]
> Sounds like we're good on the rate limiting side!
> 
> I'm guessing you probably used an old PR as an example to put these new badges together. Unfortunately, a few things have changed since then, in particular how we build examples (see #9285). You'll need to update those bits. I'd suggest either referring to [our tutorial](https://github.com/badges/shields/blob/master/doc/TUTORIAL.md) for up-to-date documentation, or using one of our existing social services as inspiration. 😉

Thanks for the feedback, @PyvesB!

I’ve updated the Bluesky services to align with the latest documentation, let me know if there’s anything else I should change.

**Comment by chitvs** [comment]
> Thanks for the updates! We're almost there!

Got it. I’ve updated the posts tests to also handle the case where a user has 0 posts, just to cover that edge case. Everything should be good now.

File: services/bluesky/bluesky-followers.tester.js [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+  label: 'followers',
+  message: isMetric,
+})
 
 t.create('User not found')
   .get('/this-user-should-not-exist-xyz123.json')
   .expectBadge({
     label: 'followers',
-    message: Joi.allow('user not found', 'invalid'),
+    message: Joi.allow('user not found'),
```
Comment (PyvesB) [inline services/bluesky/bluesky-followers.tester.js:16]: I don't think you need to wrap in `Joi.allow` anymore in this test (and its posts counterpart), you can just put the string directly.
